app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError, NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(f"{visitor["name"]}"
+                                     f" is not vaccinated.")
+
+        expiration_date = visitor["vaccine"]["expiration_date"]
+        if expiration_date and expiration_date < datetime.date.today():
+            raise OutdatedVaccineError(f"{visitor["name"]}'s"
+                                       f" vaccine is expired")
+
+        if visitor["wearing_a_mask"] is False:
+            raise NotWearingMaskError(f"{visitor["name"]}"
+                                      f" is not wearing a mask")

```

Comment:
Again, the f-string syntax is incorrect. Use single quotes around the dictionary key `name`. Change `f"{visitor["name"]}"` to `f"{visitor['name']}"`.